doc/source/tracking/paper.md

```diff
@@ -0,0 +1,136 @@
+---
+title: 'icepyx'
+tags:
+  - Python
+  - ICESat-2
+  - LiDAR
+  - elevation
+  - community
+  - cloud
+authors:
+  - name: Jessica Scheick^[Corresponding author] # note this makes a footnote
+    orcid: 0000-0002-3421-4459
+    affiliation: 1
+  - name: Wei Ji Leong
+    orcid: 0000-0003-2354-1988
+    affiliation: 2
+  - name: Kelsey Bisson
+    affiliation: 3
+  - name: Anthony Arendt
+    affiliation: 4
+  - name: Shashank Bhushan
+    affiliation: 4
+  - name: Zach Fair
+    affiliation: 5
+  - name: Raphael Hagen
+    affiliation: 4
+  - name: Sarah Hall
+    affiliation: 1  
+  - name: Scott Henderson
+    affiliation: 4
+  - name: Tom Johnson
+    affiliation: 6
+  - name: Friedrich Knuth
+    affiliation: 7
+  - name: Tian Li
+    affiliation: 8
+  - name: Zheng Liu
+    affiliation: 4
+  - name: Luis Lopez
+    affiliation: 9
+  - name: Romina Piunno
+    affiliation: 10
+  - name: Nitin Ravinder
+    affiliation: 11
+  - name: Landung "Don" Setiawan
+    affiliation: 4
+  - name: Trey Stafford
+    affiliation: 9
+  - name: Amy Steiker
+    affiliation: 9
+  - name: Tyler Sutterley
+    orcid: 0000-0002-6964-1194
+    affiliation: 4
+  - name: JP Swinski
+    affiliation: 12
+  - name: Bruce Wallin
+    affiliation: 9
+  - name: Learn2Pheonix
+    affiliation: 13
+
+# Note: first three authors are driving the publication. Additional contributors/authors are listed in alphabetical order by last name. Anyone who also contributes substantially to preparing the JOSS submission will be moved into ABC order after the first three and before the non-publication contributors to icepyx. Non-responsive coauthors will be removed from the list since their permission to be included was not granted.
+
+affiliations:
+ - name: University of New Hampshire, USA
+   index: 1
+ - name: The Ohio State University, USA
+   index: 2
+ - name: University of Colorado Boulder, USA
+   index: 3
+ - name: Institution, USA
+   index: 4
+ - name: University of Washington, USA
+   index: 5
+ - name: Institution, USA
+   index: 6
+ - name: Institution, USA
+   index: 7
+ - name: Institution, USA
+   index: 8
+ - name: Institution, USA
+   index: 9
+ - name: University of Toronto, Canada
+   index: 10
+ - name: Institution, USA
+   index: 11
+ - name: Institution, USA
+   index: 12
+ - name: Institution, USA
+   index: 13
+date: 1 September 2022
+bibliography: icepyx_pubs.bib
+
+# Optional fields if submitting to a AAS journal too, see this blog post:
+# https://blog.joss.theoj.org/2018/12/a-new-collaboration-with-aas-publishing
+aas-doi:
+aas-journal:
+---
+
+# Summary
+
+icepyx is both a software library and a community composed of ICESat-2 data users, developers, maintainers, and the scientific community.
```

Comment:
Do we need a citation for ICESat-2? E.g.:

- Markus, T., Neumann, T., Martino, A., Abdalati, W., Brunt, K., Csatho, B., Farrell, S., Fricker, H., Gardner, A., Harding, D., Jasinski, M., Kwok, R., Magruder, L., Lubin, D., Luthcke, S., Morison, J., Nelson, R., Neuenschwander, A., Palm, S., … Zwally, J. (2017). The Ice, Cloud, and land Elevation Satellite-2 (ICESat-2): Science requirements, concept, and implementation. Remote Sensing of Environment, 190, 260–273. https://doi.org/10.1016/j.rse.2016.12.029

Or maybe there's a more suitable one. Just occurred to me that there's no mention of what ICESat-2 actually does in this entire paper so far :joy:

Reply:
Good point. I added a short paragraph on IS2 and this ref. Thanks!